web-client/src/presenter/actions/TrialSession/updateTrialSessionAction.js

```diff
@@ -17,26 +17,41 @@ export const updateTrialSessionAction = async ({
   path,
   props,
 }) => {
-  const startDate = // AAAA-BB-CC
-    (props.computedDate &&
+  const startDate =
```

Comment:
💭 Potential DOD could be to try and DRY up this and `web-client/src/presenter/actions/TrialSession/createTrialSessionAction.js`

Reply:
Pulled out the shop vac and DRYed up!